README.md

```diff
@@ -17,26 +17,14 @@
   under the License.
 -->
 
-## Apache Arrow
-
-<table>
-  <tr>
-    <td>Build Status</td>
-    <td>
-    <a href="https://travis-ci.org/apache/arrow">
-    <img src="https://travis-ci.org/apache/arrow.svg?branch=master" alt="travis build status" />
-    </a>
-    </td>
-    <td>Code Coverage</td>
-    <td>
-    <a href="https://codecov.io/gh/apache/arrow">
-    <img src="https://codecov.io/gh/apache/arrow/branch/master/graph/badge.svg" alt="codecov.io code coverage" />
-    </a>
-    </td>
-  </tr>
-</table>
-
-### Powering In-Memory Analytics
+# Apache Arrow
+
+[![Build Status](https://travis-ci.org/apache/arrow.svg?branch=master)](https://travis-ci.org/apache/arrow)
+[![Coverage Status](https://codecov.io/gh/apache/arrow/branch/master/graph/badge.svg)](https://codecov.io/gh/apache/arrow?branch=master)
+[![License](http://img.shields.io/:license-Apache%202-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0.txt)
```

Comment:
We should link to our license file, which attributes third party licenses that are utilized in the codebase

Reply:
Makes sense. 👍